core/domain/topic_domain.py

```diff
@@ -250,6 +341,26 @@ def validate(self):
                 'Expected description to be a string, received %s'
                 % self.description)
 
+        if not isinstance(self.subtopics, list):
+            raise utils.ValidationError(
+                'Expected subtopics to be a list, received %s'
+                % self.subtopics)
+
+        if not isinstance(self.schema_version, int):
+            raise utils.ValidationError(
+                'Expected schema version to be an integer, received %s'
```

Comment:
check that it's less than CURRENT_SUBTOPICS_SCHEMA_VERSION and bigger than 0 or 1 (not sure what your starting number is)

(in general, check everything you could possibly check without fetching additional stuff from datastore)

Reply:
Done.